lib/napa/output_formatters/representer.rb

```diff
@@ -1,11 +1,20 @@
 require 'roar/decorator'
-require 'roar/representer/json'
-require 'roar/representer/feature/coercion'
+require 'roar/version'
+
+if Roar::VERSION >= '1.0.0'
+  require 'roar/json'
+  require 'roar/coercion'
+  REPRESENTER_JSON_MODULE = Roar::JSON
```

Comment:
This will set a global constant in any app that uses Napa.

Any reason you can't pull this conditional around the `include` within the `Representer` class?